docs/Frietjes.md

```diff
@@ -0,0 +1,13 @@
+### Ingredients
+(loskokende) aardappel 1 kg
```

Comment:
Perhaps we can increase the quantity?

Reply:
Always, the more the merrier. I'll double the amount of potatoes for french fries.